docker-compose.yaml

```diff
@@ -1,27 +1,24 @@
-version: '3'
-
 services:
-  brapi-java-server-v1:
-    image: brapicoordinatorselby/brapi-java-server:v1
-    restart: unless-stopped
-    depends_on:
-      - postgres
-    volumes:
-      - /home/jenkins/brapi.org/brapi-test-server/properties/v1/application.properties:/home/brapi/properties/application.properties
   brapi-java-server-v2:
-    image: brapicoordinatorselby/brapi-java-server:v2
-    restart: unless-stopped
+    # Update this image with whatever image you have access to
+    image: brapicoordinatorselby/brapi-java-server:latest
+    ports:
+      - "8080:8080"
+      # Expose DEBUG port
+      - "5005:5005"
```

Comment:
8080 should not be exposed for the production server, all traffic should go through the rev-proxy service

5005 probably shouldn't be exposed... I'm not sure. It could be handy to have a public debug port open, but it would technically be unprotected access to live code ...

Reply:
We should see if there is a way to only expose 5005 through an ssh tunnel or something similar. I don't know how to do that off the top of my head, maybe a project for another day

Reply:
I removed these.